text/2532-associated-type-defaults.md

```diff
@@ -351,7 +351,7 @@ in other items specified in the implementation.
 If an implementation does make the associated type available for
 further specialization, then other definitions in the implementation
 may not assume the given underlying specified type of the associated type
-and may only assume that it is `Self::TheAsociatedType`.
+and may only assume that it is `Self::Bar`.
```

Comment:
This is not a typo; `TheAssociatedType` does not refer to `Self::Bar` but the "that" associated type referred to throughout the paragraph beginning with "an associated type".

Reply:
Ah, that makes sense, they're different types of course (trait default vs. impl default). Fixed.